src/src/ex7/client/src/redux/reducers/items-entities-reducer.js

```diff
@@ -0,0 +1,68 @@
+import actionTypes from "../actions/constants";
+
+const initialState = {
+  tasksStatus:true,
+  searchInput: "",
+  items: [],
+};
+
+const itemsEntitiesReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case actionTypes.ADD_ITEMS:
+      return {
+        ...state,
+        items: [...state.items, ...action.payload],
```

Comment:
instead of generic `payload` variable on top of `action` variable, better pass the right prop name such as `action.payload.items` in this case as payload is too generic name and may cause bugs and confusion